src/pages/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,32 @@
+import { Footer } from "../../components/Footer/Footer";
+import { NavBar } from "../../components/NavBar/NavBar";
+import { useUser } from "../../contexts/UserContext";
+import { ProductCard } from "../../components/ProductCards/ProductCard";
+
+export const Wishlist = () => {
+  const { userState, userDispatch } = useUser();
+
+  return (
+    <div className="wrapper justify-between">
+      <NavBar />
+      <main className="flex-1">
+        <div className="container">
+          <div className="h3 txt-center">My Wishlist</div>
+          <hr />
+          <div className="flex flex-wrap justify-center">
+            {userState.userWishlist.length === 0 ? (
+              <div className="width-100 mg-y-2 txt-center">
+                No items added in wishlist...
```

Comment:
suggestion : here you can add a "Go to products page" button if wishlist is empty

Reply:
Thanks for suggesting this. Added this button on wishlist page if its empty.